internal/restclient_impl.go

```diff
@@ -23,6 +23,18 @@ type restClientImpl struct {
 	disgo api.Disgo
 }
 
+func (r *restClientImpl) GetGuildStickers(guildID api.Snowflake) ([]*api.MessageSticker, restclient.RestError) {
+	panic("implement me")
```

Comment:
Once @TopiSenpai finishes the refactor we can impl this into the Routes. It seems worthless to make changes to the other repo atm